examples/energy-management-app/energy-management-common/src/EVSEManufacturerImpl.cpp

```diff
@@ -104,6 +103,223 @@ CHIP_ERROR EVSEManufacturer::Shutdown()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR FindNextTarget(const uint8_t dayOfWeekMap, uint16_t minutesPastMidnightNow_m, uint16_t & targetTimeMinutesPastMidnight_m,
+                          DataModel::Nullable<Percent> & targetSoC, DataModel::Nullable<int64_t> & addedEnergy_mWh,
+                          bool bAllowTargetsInPast)
+{
+
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    EnergyEvse::Structs::ChargingTargetScheduleStruct::Type entry;
+
+    uint16_t minTimeToTarget_m = 1440; // 24 hours
+    bool bFound                = false;
+
+    EVSEManufacturer * mn = GetEvseManufacturer();
+    VerifyOrReturnError(mn != nullptr, CHIP_ERROR_UNINITIALIZED);
+
+    EnergyEvseDelegate * dg = mn->GetEvseDelegate();
+    VerifyOrReturnError(dg != nullptr, CHIP_ERROR_UNINITIALIZED);
+
+    const DataModel::List<const EnergyEvse::Structs::ChargingTargetScheduleStruct::Type> & chargingTargetSchedules =
+        dg->GetEvseTargetsDelegate()->GetTargets();
+    for (auto & chargingTargetScheduleEntry : chargingTargetSchedules)
+    {
+        if (chargingTargetScheduleEntry.dayOfWeekForSequence.GetField(static_cast<TargetDayOfWeekBitmap>(0x7F)) & dayOfWeekMap)
```

Comment:
0x7F should probably be a named constant.

Reply:
A constant is now used.